lib/core/request.js

```diff
@@ -231,6 +231,32 @@ class Request {
     return this[kHandler].onResponseStarted?.()
   }
 
+  onField (name, value) {
+    assert(!this.aborted)
+    assert(!this.completed)
+
+    if (this[kHandler].onField) {
+      try {
+        return this[kHandler].onField(name, value)
```

Comment:
Shouldn't this be equivalent, or are you looking to avoid create the `try/catch` scope?

```suggestion
        return this[kHandler].onField?.(name, value)
```

Reply:
Avoiding the try catch